packages/dma-contracts/contracts/libs/UseStorageSlot.sol

```diff
@@ -41,6 +41,21 @@ library StorageSlot {
     return values;
   }
 
+  /**
+   * @dev Clears the stored array at the specified slot position.
+   * @param slotPosition The position of the storage slot.
+   */
+  function clearStoredArray(bytes32 slotPosition) internal {
```

Comment:
I think we only need to set the length to 0 here. Doing a loop like that could be very costly. I don't think we are concerned about the values in the transient storage being read/leaked, but rather that the transient storage starts from zero every time.

Also this could be prevented by having a proper reentrancy lock in `executeOp`

Reply:
correct. I was too fast with this one